src/Controller/AventController.php

```diff
@@ -10,69 +10,77 @@
 class AventController extends AbstractController
 {
     private $twig;
+    protected $futureEnabled;
+    protected $enabledYears = array('2013', '2017', '2019');
 
     public function __construct(Environment $twig)
     {
         $this->twig = $twig;
+        if('prod' != getenv('APP_ENV')) {
```

Comment:
This is not a good idea, use the parameter `kernel.environment` by injecting it! (notice that `getenv()` should never be used)

Reply:
fixed